### PR TITLE
Fix GroundPrimitive documentation

### PR DESCRIPTION
**Pull request by TomPed**
For #3214

### CODE REVIEW

**Comment by lilleyse** [comment]
Thanks, merged.

File: Source/Scene/GroundPrimitive.js [hunk 1]
```diff
@@ -74,6 +74,12 @@ define([
      * is supported at this time.
      * </p>
      * <p>
+     * Because of the cutting edge nature of this feature in WebGL, it requires the EXT_frag_depth extension, which is currently only supported in Chrome,
+     * Firefox, and Edge. Apple support is expected in iOS 9 and MacOS Safari 9. Android support varies by hardware and IE11 will most likely never support
```
Comment (pjcozzi) [inline Source/Scene/GroundPrimitive.js]: Isn't iOS 9 already out?

Comment (TomPed) [inline Source/Scene/GroundPrimitive.js]: Looks like it is. I also believe that Safari 9 is out as well.

**Comment by lilleyse** [comment]
Oh true... both are out. Have we checked if the extension is supported for iOS and Safari?

File: Source/Scene/GroundPrimitive.js (hunk 1 above)
Comment (lilleyse) [inline Source/Scene/GroundPrimitive.js]: Oh true... both are out. Have we checked if the extension is supported for iOS and Safari?

Comment (pjcozzi) [inline Source/Scene/GroundPrimitive.js]: No, I will not.

Comment (pjcozzi) [inline Source/Scene/GroundPrimitive.js]: hehe, I mean I have not.

Comment (mramato) [inline Source/Scene/GroundPrimitive.js]: There's also no need to mention the Entity API either.  I would change this doc to something like:

> This feature requires the the WebGL EXT_frag_depth extension, which is supported in Chrome, Firefox, Safari, and Edge; but not IE 11. For mobile hardware, EXT_frag_depth support may also vary by device, you can use webglreport.com to verify support for your target hardware.

I think I tested it on Safari, but if you guys want to check again, go for it.

Comment (shunter) [inline Source/Scene/GroundPrimitive.js]: WebGLReport from iOS 9.2 on my iPhone 6.  No frag_depth.

Platform:	iPhone
Browser User Agent:	Mozilla/5.0 (iPhone; CPU iPhone OS 9_2 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13C75 Safari/601.1
Context Name:	webgl
GL Version:	WebGL 1.0 (OpenGL ES 2.0 Apple A8 GPU - 75.11.5)
Shading Language Version:	WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.00)
Vendor:	WebKit
Renderer:	WebKit WebGL
Antialiasing:	Not available
ANGLE:	No
Major Performance Caveat:	Not implemented
Vertex Shader
Max Vertex Attributes:	16
Max Vertex Uniform Vectors:	128
Max Vertex Texture Image Units:	8
Max Varying Vectors:	8
Best float precision:	[-2127, 2127] (23)
Transform Feedback
Coming in WebGL 2
Rasterizer
Aliased Line Width Range:	[1, 16]
Aliased Point Size Range:	[1, 511]
Fragment Shader
Max Fragment Uniform Vectors:	64
Max Texture Image Units:	8
float/int precision:	highp/highp
Best float precision:	[-2127, 2127] (23)
Framebuffer
Max Color Buffers:	1
RGBA Bits:	[8, 8, 8, 8]
Depth / Stencil Bits:	[24, 8]
Max Render Buffer Size:	4096
Max Viewport Dimensions:	[4096, 4096]
Textures
Max Texture Size:	4096
Max Cube Map Texture Size:	4096
Max Combined Texture Image Units:	8
Max Anisotropy:	16
Uniform Buffers
Coming in WebGL 2
Supported Extensions:
EXT_blend_minmax
EXT_sRGB
OES_texture_float
OES_texture_float_linear
OES_texture_half_float
OES_texture_half_float_linear
OES_standard_derivatives
EXT_shader_texture_lod
WEBKIT_EXT_texture_filter_anisotropic
OES_vertex_array_object
OES_element_index_uint
WEBGL_lose_context
WEBKIT_WEBGL_compressed_texture_pvrtc
WEBGL_depth_texture
ANGLE_instanced_arrays

Comment (mramato) [inline Source/Scene/GroundPrimitive.js]: That sucks.  I just confirmed with @klim705 that frag depth is supported on desktop Safari.  I'll check my iPad when I get home tonight.

**Pull request merged**